webportal/README.md

```diff
@@ -1,15 +1,89 @@
 # Web Portal
 
-The web portal is the entrance for job and cluster management.
-User can submit, monitor, or kill the job through the web UI.
-Cluster operator can also see and manage cluster state through web portal. 
+## Goal
 
-## Deployment
+The web portal is the entrance for job and cluster management. User can submit, monitor, or kill the job through the web UI. Cluster operator can also see and manage cluster state through web portal.
+
+## Architecture
+
+An [express](https://expressjs.com/) served, [AdminLTE](https://adminlte.io/) themed, and [EJS](http://ejs.co/) rendered static web application.
+
+## Dependencies
+
+Since [job toturial](../docs/job_tutorial.md) is included in the document tab, make sure **`docs`** directory is exists as a sibling of `web-portal` directory.
+
+To run web portal, the following services should be started, and url of services should be correctly configured:
+
+* [REST Server](../rest-server)
+* [Prometheus](../prometheus)
+* [Grafana](../grafana)
+* YARN
+* Kubernetes
+
+## Build
+
+*In current design, there is nothing to do in build stage.*
```

Comment:
show this service image build and push is better
https://github.com/Microsoft/pai/tree/dff283ae0292141790d87a4c42d75d62cfee81da/pai-management/src/webportal

Reply:
@YanjieGao Should I assume that readers of this readme will deploy standalone or by pai-management, or both? In fact I'm not clear about this.

Reply:
@sterowang , bin, qixiang's question need your help